R/get_available_datasets.R

```diff
@@ -1,14 +1,21 @@
-#' Get supported data sets
+#' Fetch of build table of supported data sets
 #'
-#' @description The function searches the environment for R6 class objects and
-#' extracts the summary information from the available classes using
-#' their `summary` methods. In practice this means that it can be used
-#' to indicate supported data sets.
+#' @description The function returns data on what countries are available from
+#' the data provided with this package. If render = TRUE The function searches
```

Comment:
suggest reword to data provided with this package either using a cached dataset or built by searching the target namespace. (and then shift the docs about render to that param)